brackend/db/models.py

```diff
@@ -45,6 +58,42 @@ class UserTournament(Base):
     id = Column(Integer, primary_key=True)
     user_id = Column(Integer, ForeignKey("users.id"))
     tournament_id = Column(Integer, ForeignKey("tournaments.id"))
+    is_organizer = Column(Boolean, default=False)
```

Comment:
Any chance we have other roles for this? Potentially like a spectator? Might want to go with a `role` enum here, with values `PLAYER` or `ORGANIZER` so we can always add more later